kornia/augmentation/augmentation.py

```diff
@@ -257,10 +312,15 @@ def generate_parameters(self, batch_shape: torch.Size) -> Dict[str, torch.Tensor
         return dict()
 
     def compute_transformation(self, input: torch.Tensor, params: Dict[str, torch.Tensor]) -> torch.Tensor:
-        return F.compute_intensity_transformation(input)
```

Comment:
Can we make this to happen automatically ? If the user doesn’t specify, then Internally returns identity

Reply:
Having this is to make the users aware of what to do for transformations.